### ISSUE TITLE
[Training]learning rate

### ISSUE THREAD

**Issue by davidingram123** [KAIST-AILab/SyncVSR#23]
@snoop2head Hi, sorry to trouble you again. I tried training, but I found that the learning rate changes differently from what was shown in the experiment you provided earlier : [wandb report](https://wandb.ai/quoqa-nlp/cross-modal-sync/reports/LRW-Training-Log--Vmlldzo5Nzc5MDQ4?accessToken=b1o90x4tiiq1tvttq7hl1re31ts05njoq18sfmb6haak784en6lpu05zzc46you4) .
What could be the reason for this?[test](https://wandb.ai/daiyingjie2024-/cross-modal-sync/reports/Untitled-Report--Vmlldzo5OTQ0ODIz?accessToken=ba16lj0nagt1yt2ihmoan259tqnhtnpyt8tnk2v2urtxmn1c61u9udq6dmh6kpeu)

**Comment by snoop2head** [comment]
Hello @davidingram123,
I think you should adjust batch size or number of training step size to match 200 epochs! 
But as long as the peak learning rate is the same, matching 200 epochs will resolve related issue.

**Comment by davidingram123** [comment]
@snoop2head thank you
